holocron/trainer/core.py

```diff
@@ -339,7 +339,7 @@ def evaluate(self) -> Dict[str, float]:
 
         self.model.eval()
 
-        val_loss, top1, top5, num_samples = 0., 0, 0, 0
+        val_loss, top1, top5, num_samples = 0., 0., 0., 0
```

Comment:
I'm curious: was there some necessary casting here? I thought in python 3, the division of integer was casted to float automatically

Reply:
Yes agree, I don't remeber why I made this change ..